packages/synapse-react-client/src/components/CardDeck/CardDeck.stories.ts

```diff
@@ -32,23 +32,23 @@ export const CCPKResourcesDemo: Story = {
   },
 }
 
-export const ProgramsDemo: Story = {
-  args: {
-    entityId: 'syn17024173',
-    titleColumnName: 'Program',
-    descriptionColumnName: 'Short Description',
-    ctaButtonTextColumnName: 'ctaButtonText',
-    ctaButtonURLColumnName: 'ctaButtonURL',
-    titleIconFileHandleColumnName: 'Homepage Image',
-    headerImageFileHandleColumnName: undefined,
-    linkConfig: {
-      isMarkdown: false,
-      baseURL: 'Explore/CardDeck/DetailsPage',
-      URLColumnName: 'title',
-      matchColumnName: 'title',
-    },
-  },
-}
+// export const ProgramsDemo: Story = {
+//   args: {
+//     entityId: 'syn17024173',
+//     titleColumnName: 'Program',
+//     descriptionColumnName: 'Short Description',
+//     ctaButtonTextColumnName: 'ctaButtonText',
+//     ctaButtonURLColumnName: 'ctaButtonURL',
+//     titleIconFileHandleColumnName: 'Homepage Image',
+//     headerImageFileHandleColumnName: undefined,
+//     linkConfig: {
+//       isMarkdown: false,
+//       baseURL: 'Explore/CardDeck/DetailsPage',
+//       URLColumnName: 'title',
+//       matchColumnName: 'title',
+//     },
+//   },
+// }
```

Comment:
Why is this commented out?

Reply:
It's my understanding from our conversation yesterday that we won't pursue more variants in the CardDeckV1 component so I'm not continuing to develop the variation that would capture the functionality of the programs component.

Hah I was feeling a little attached I guess and didn't want to fully delete but now that you ask I think deletion makes more sense.